managedtenants/utils/ocm.py

```diff
@@ -159,6 +159,83 @@ def disable_addon(self, addon_id):
             f"/api/clusters_mgmt/v1/addons/{addon_id}", json={"enabled": False}
         )
 
+    def get_migrations(self):
+        return self._get("/api/clusters_mgmt/v1/addon_migrations")
+
+    def get_migration(self, addon_id):
+        return self._get(f"/api/clusters_mgmt/v1/addon_migrations/${addon_id}")
+
+    def post_addon_migration(self, addon_id):
```

Comment:
get_migration vs post_addon_migration... let's use the same naming. either:

- get_migration and post_migration
or
- get_addon_migration and post_addon_migration